### PR TITLE
Add heroic lab to home page

### PR DESCRIPTION
**Pull request by coppolaemilio**
This PR adds the logo from Heroic Labs, a platinum sponsor, to our home page:
image

### CODE REVIEW

File: pages/home.html [hunk 1]
```diff
@@ -552,4 +555,4 @@ <h2 class="text-center">Sponsored by</h2>
 	});
 </script>
 
-{% include footer.html %}
+{% include footer.html %}
```
Comment (YuriSizov) [inline pages/home.html]: Needs a new line at the end of the file, which Akien added in his recent PR, and this PR removes again.

File: pages/home.html [hunk 2]
```diff
@@ -510,7 +513,7 @@ <h2 class="text-center">Sponsored by</h2>
 			game: '{{ project.title }}',
 				by: '{{ project.author }}',
 					url: '{{ project.url }}',
-						image: '{{ project.image }}',
+						image: '{{ project.gallery[0] }}',
```
Comment (YuriSizov) [inline pages/home.html]: Was that intentional?

File: pages/home.html (hunk 1 above)
Comment (coppolaemilio) [reply pages/home.html]: Missed that one! I did not update my local `master` when creating the branch. Should be good now 👍

File: pages/home.html (hunk 2 above)
Comment (coppolaemilio) [reply pages/home.html]: Right now in this branch the `.image` and the `.gallery[0]` is the same image. This is more relevant on the other branch which will take a new cover image and use the first image from the gallery for the hero backgrounds. I'll move it to the other PR

**Review by YuriSizov — APPROVED** [review]

**Comment by YuriSizov** [comment]
Thanks!

**Pull request merged**